<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOrderDialog</class>
 <widget class="QDialog" name="AddOrderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>458</width>
    <height>552</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Buat Order</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Tentukan Konsumen</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCustomer">
         <property name="toolTip">
          <string>Konsumen Baru</string>
         </property>
         <property name="text">
          <string>Baru</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Data Order</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="1,3">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Nama</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Tanggal Order</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxProduct">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Produk</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Deal Harga</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Total</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Qty</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="1" alignment="Qt::AlignRight">
         <widget class="QPushButton" name="pushButtonStore">
          <property name="text">
           <string>Tambahkan</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxQ">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Ukuran</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2,0,1,2,0">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>P</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBoxP">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <double>999.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.500000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>T</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBoxT">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <double>999.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.500000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="spinBoxH">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
          <property name="singleStep">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateTimeEdit" name="dateTimeEdit">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2024</year>
            <month>5</month>
            <day>4</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>dddd, dd MMMM yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>15</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>15</number>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButtonReset">
          <property name="text">
           <string>Reset</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSave">
          <property name="toolTip">
           <string>Konsumen Baru</string>
          </property>
          <property name="text">
           <string>Simpan</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonBayar">
          <property name="text">
           <string>Bayar</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionDeleteSelectedItems">
   <property name="text">
    <string>Hapus terpilih</string>
   </property>
   <property name="toolTip">
    <string>Hapus item terpilih</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButtonCustomer</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>comboBoxProduct</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>doubleSpinBoxP</tabstop>
  <tabstop>doubleSpinBoxT</tabstop>
  <tabstop>spinBoxH</tabstop>
  <tabstop>doubleSpinBoxQ</tabstop>
  <tabstop>pushButtonStore</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButtonReset</tabstop>
  <tabstop>pushButtonSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
